package-configuration/linux-cli/test_get_cloudify.py

```diff
@@ -144,6 +162,30 @@ def test_check_cloudify_installed_in_venv(self):
         finally:
             shutil.rmtree(tmp_venv)
 
+    def test_get_requirements_from_source_url(self):
+        def get(url, destination):
+            return self._create_dummy_requirements_tar(destination)
+
+        self.get_cloudify.download_file = get
```

Comment:
since we already use mock here, you can use mock.patch:

```
@mock.patch('get_cloudify.download_file', side_effect= _create_dummy_requirements_tar)
def test_get_requirements_from_source_url(self, mock_download_file):
   installer = self.get_cloudify.CloudifyInstaller()
   req_list = installer._get_default_requirement_files('null')
   self.assertEquals(len(req_list), 1)
   self.assertIn('dev-requirements.txt', req_list[0])
```

* didn't check but I think it's close :wink: